site/posts/pandas-tutorial.ipynb

```diff
@@ -0,0 +1,59 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "sufficient-document",
+   "metadata": {},
+   "source": [
+    "The NCAR/UCAR virtual Python Tutorial Seminar Series continues with an introduction to the Python package `pandas` on **Wednesday, May 26th at 1 PM Mountain**. This session will be led by Drew Camron and Max Grover.\n",
+    "\n",
+    "The content for this tutorial is hosted on Max's [NCAR Pandas Tutorial Github Repository](https://github.com/mgrover1/ncar_pandas_tutorial)\n",
+    "\n",
+    "## Preparation\n",
+    "\n",
+    "- Check your conda install if this is the first tutorial in the series you are joining us for with `conda --version` and if necessary [install by following these instructions](https://docs.conda.io/en/latest/miniconda.html).\n",
+    "\n",
+    "<!-- TEASER_END -->\n",
+    "\n",
+    "- Navigate to Maxs [NCAR Pandas Tutorial Github Repository](https://github.com/mgrover1/ncar_pandas_tutorial).\n",
+    "\n",
+    "- Clone this repository by clicking the green \"Code\" button and copying the SSH address under the word \"Clone\". This can be done by clicking the clipboard icon.\n",
+    "\n",
+    "- In your terminal, type `git clone PASTE` where you paste the SSH address you just copied (`git clone git@github.com:mgrover1/ncar_pandas_tutorial.git`)\n",
+    "\n",
+    "- Create a Conda environment from the `conda_environment.yml` file with the terminal command `conda env create -f conda_environment.yml`. Make sure you are in the local `ncar_pandas_tutorial` directory for this step.\n",
```

Comment:
Going to use environment.yml - keeping it very basic (just pandas and jupyterlab)

Reply:
That's fine! Thanks for reviewing it :)